man/deepsea-stage-3.7

```diff
@@ -0,0 +1,174 @@
+.TH deepsea 7
+.SH NAME
+DeepSea stage 3 \- a deeper insight
+.SH DESCRIPTION
+.B DeepSea Stage 3 (Deploy)
+is a Salt orchestration that deploys the core services of ceph
+.RE
+.PD
+.SH Preface
+.PP
+After configuration and setup stages have been completed successfully we are ready to actually deploy and install ceph.
+
+.B Validation:
+.RS
+DeepSea performs a set of system validation tests before running the ramainder of the stage.
+
+* DEV_ENV
+  This option tells you if a Developer Environment is expected. Setting this option bypasses some validations that 
+  would lead to insufficient replication or data insafety.
+  Do not set this option in production.
+
```

Comment:
Might want to mention that DEV_ENV can be set as either an environment variable or pillar variable.  

Some rewording:

Set this option to 'true' for development environments.  Some validations will be bypassed such as requiring minimums for Ceph monitors and storage nodes.